data/experiment-recipe-samples/android-96-multifeature.json

```diff
@@ -0,0 +1,87 @@
+{
+  "appId": "org.mozilla.firefox",
+  "appName": "fenix",
+  "application": "org.mozilla.firefox",
+  "arguments": {},
+  "branches": [
+    {
+      "features": [
+        {
+          "enabled": false,
+          "featureId": "fenix-default-browser",
+          "value": {}
+        },
+        {
+          "enabled": true,
+          "featureId": "homescreen",
+          "value": {
+            "sections-enabled": {
+              "jumpBackIn": false,
+              "pocket": false,
+              "recentExplorations": false,
+              "recentlySaved": false,
+              "topSites": true
+            }
+          }
+        }
+      ],
+      "ratio": 1,
+      "slug": "control"
+    },
+    {
+      "features": [
+        {
+          "enabled": true,
+          "featureId": "fenix-default-browser",
+          "value": {}
+        },
+        {
+          "enabled": true,
+          "featureId": "homescreen",
+          "value": {
+            "sections-enabled": {
+              "jumpBackIn": true,
+              "pocket": true,
+              "recentExplorations": true,
+              "recentlySaved": true,
+              "topSites": true
+            }
+          }
+        }
+      ],
+      "ratio": 1,
+      "slug": "treatment"
+    }
+  ],
+  "bucketConfig": {
+    "count": 10000,
+    "namespace": "firefox-android-multifeature-test",
```

Comment:
I remember the namespace for multifeature experiments being different, a concatenation of the features used being part of it. Has that changed and if not, will this affect testing?

Reply:
No that only matters for trying to enroll in multiple multi-feature experiments, I wouldn't worry about it here.